go/app/server_test.go

```diff
@@ -123,8 +164,9 @@ func TestAddItem(t *testing.T) {
 				"category": "phone",
 			},
 			injector: func(m *MockItemRepository) {
-				// STEP 6-3: define mock expectation
-				// succeeded to insert
+				m.EXPECT().
+					Insert(gomock.Any(), gomock.Any()).
+					Return(nil)
```

Comment:
[nits]
厳密に何回呼ばれるかも確認すると良さそうです！
ref: https://pkg.go.dev/go.uber.org/mock@v0.5.0/gomock#Call.Times
他の場所も同様ですね

```suggestion
				m.EXPECT().
					Insert(gomock.Any(), gomock.Any()).
					Return(nil).Times(1)
```